src/main/java/org/sagebionetworks/bridge/hibernate/HibernateStudy.java

```diff
@@ -20,7 +21,7 @@
 @BridgeTypeName("Study")
 public class HibernateStudy implements Study {
     @Id
-    private String id;
+    private String identifier;
```

Comment:
If the attribute is still called "id" in the database, how does Hibernate know to look for "id" instead of "identifier"?

Reply:
I don't know how, but I can verify it does (maybe because it has the schema information on the primary key from the table). For clarity, I'll add a @Column annotation to HibernateStudy.